### PR TITLE
Matrix operation in C

### LINKED ISSUE #1122: Matrix operation in C
1.Matrix_Lower_triangular
2.Matrix_Upper_triangular
3.Matrix_Sort
4.Matrix_Multiplication

### PR DESCRIPTION
**Pull request by somya-kapoor**
Fixes #1122

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Sample Input / Output.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

#### Changes proposed in this pull request:

- Added Matrix operation in C (Matrix_Lower_triangular,Matrix_Upper_triangular,Matrix_Sort
    Matrix_Multiplication)

#### Languages Used:
- C

#### Files Added:
- Matrix_Lower_triangular.c
- Matrix_Multiplication.c
- Matrix_Sort.c 
- Matrix_Upper_triangular.c

### CODE REVIEW

File: Matrix_Operations/C/Matrix_Lower_triangular.c [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    printf("Enter the Matrix Elements: \n");
+    for(i = 0; i < row; i++)
+    {
+        for(j = 0; j < col; j++)
+        {
+            scanf("%d", &matrix[i][j]);
+        }
+    }
+ 
+    //Lower triangular matrix is a matrix which contain elements below
```
Comment (SakshiShreya) [inline Matrix_Operations/C/Matrix_Lower_triangular.c]: Please add a space after `//`

File: Matrix_Operations/C/Matrix_Multiplication.c [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+                         {4, 5, 6}, 
+                         {7, 8, 9}};                        
+  
+    multiply(matrix1, matrix2, result); 
+  
+    printf("Result matrix is \n"); 
+    for (i = 0; i < N; i++) 
+    { 
+        for (j = 0; j < N; j++) 
+            printf("%d ",result[i][j]);
```
Comment (SakshiShreya) [inline Matrix_Operations/C/Matrix_Multiplication.c]: Add a space after `,`

File: Matrix_Operations/C/Matrix_Upper_triangular.c [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    printf("Enter the Matrix Elements: \n");
+    for(i = 0; i < row; i++)
+    {
+        for(j = 0;j < col;j++)
+        {
+            scanf("%d", &matrix[i][j]);
+        }
+    }
+
+    //form upper triangular marix
```
Comment (SakshiShreya) [inline Matrix_Operations/C/Matrix_Upper_triangular.c]: Add space after `//` in all comments

**Comment by somya-kapoor** [comment]
@SakshiShreya Please review.

**Pull request merged**